src/compiler/typescript_to_rust/generators/ic_object/functions/accept_message.ts

```diff
@@ -0,0 +1,13 @@
+import { Rust } from '../../../../../types';
+
+export function generateIcObjectFunctionAcceptMessage(): Rust {
+    return /* rust */ `
+        fn _azle_ic_accept_message(
+            _this: &boa_engine::JsValue,
+            _aargs: &[boa_engine::JsValue],
+            _context: &mut boa_engine::Context
+        ) -> boa_engine::JsResult<boa_engine::JsValue> {
+            Ok(ic_cdk::api::call::accept_message().azle_into_js_value(_context))
```

Comment:
This will create a JsValue::Undefined correct? I wonder if we should then make the TypeScript type return undefined